services/load/service.go

```diff
@@ -603,7 +603,7 @@ func (s *Service) removeHandlers() error {
 	for _, id := range diff(s.handlers, loadedHandlers) {
 		topicID, handlerID, found := strings.Cut(id, "/")
 		if !found {
-			return errors.New("expected id to be topicID/handlerID but it was: \"" + id + "\"")
+			return fmt.Errorf("expected id to be topicID/handlerID but it was: \"%s\"", id)
```

Comment:
non-blocking - I like `%q` over `\"%s\"` because it escapes bad characters and looks cleaner in the formatting string.